classes/template/PKPTemplateManager.inc.php

```diff
@@ -285,6 +285,9 @@ function initialize() {
 			$nmService = ServicesContainer::instance()->get('navigationMenu');
 
 			\HookRegistry::register('LoadHandler', array($nmService, '_callbackHandleCustomNavigationMenuItems'));
+
+			// Register Public library files handler
+			\HookRegistry::register('LoadHandler', array($this, 'setPublicLibraryFilesHandler'));
```

Comment:
I'm confused about this -- is it necessary?

Reply:
@asmecher It may be my incomplete understanding on how the handlers are being registered on OJS, but shouldn't I register the handler like that in order for the new URL pattern (/publicLibrary/downloadLibraryFile/id) to became available?

Reply:
No, simply putting the handler class in the right spot and adding an `index.php` wrapper for it should be enough. You should be able to remove this.

Reply:
In that case I was missing something there...